phonebuddies/PhoneBuddyManager.py

```diff
@@ -102,29 +102,49 @@ def admin_buddy_pairs(self):
 
         return [ADMIN_EMAILS[:2]]
 
+    def send_admins_without_buddy_emails(self , buddy_pairs):
+        #extract a list of email addresses from buddy_pairs
+        buddy_emails_list=[]
+        for sublist in buddy_pairs:
+            buddy_emails_list.extend(sublist)
+        
+        self.admins=ADMIN_EMAILS
+        self.are_there_any_admins_without_buddies = False
+
+        #only send to admins without buddies
+        for email in self.admins:
+            if email not in buddy_emails_list:
+                self.are_there_any_admins_without_buddies = True
+                draft = EmailDraft.draft_admins_without_buddy_emails(email)
+                self._email_sender.send_email(draft)
 
+        return(self.are_there_any_admins_without_buddies)
+        
     def emails_sent_status(self):
         return "Yes" if self._really_send_emails else "Dry-run complete"
 
-
```

Comment:
I actually prefer to have two blank lines between functions. Could you put that back here please as well as before `email_sent_status`?